lib/keychain.c

```diff
@@ -24,6 +24,7 @@
 #include "memory.h"
 #include "linklist.h"
 #include "keychain.h"
+#include "config.h"
```

Comment:
I already commented earlier, `config.h` does not belong here like this.  It must always be the first include, or `zebra.h` must be the first include (because it includes `config.h` first.)